lambdas/process_webhooks/test/test_process_webhooks.py

```diff
@@ -12,6 +12,29 @@
 
 
 class ProcessWebhooksTestCase(TestCase):
+    headers = {
+        'Content-Type': 'application/json',
+        'X-GitHub-Event': ''
+    }
+
+    @patch.dict(os.environ, {'TARGET_URL': 'http://www.example.com'})
+    def test_get_target_url_pr(self):
+        self.headers['X-GitHub-Event'] = 'pull_request'
+        url = _get_target_url(self.headers)
+        self.assertEqual(url, 'http://www.example.com/ghprbhook/')
+
+    @patch.dict(os.environ, {'TARGET_URL': 'http://www.example.com'})
+    def test_get_target_url_comment(self):
+        self.headers['X-GitHub-Event'] = 'issue_comment'
+        url = _get_target_url(self.headers)
+        self.assertEqual(url, 'http://www.example.com/ghprbhook/')
+
+    @patch.dict(os.environ, {'TARGET_URL': 'http://www.example.com'})
+    def test_get_target_url_push(self):
+        self.headers['X-GitHub-Event'] = 'push'
+        url = _get_target_url(self.headers)
+        self.assertEqual(url, 'http://www.example.com/github-webhook/')
+
```

Comment:
nit: It would be nice to add a test for the error condition (when the event type is not push, pr, or comment).